collect_app/src/main/java/org/odk/collect/android/dao/helpers/ContentResolverHelper.java

```diff
@@ -18,14 +18,14 @@
 import android.content.Context;
 import android.database.Cursor;
 import android.net.Uri;
-import android.provider.OpenableColumns;
 import android.webkit.MimeTypeMap;
 
 import org.odk.collect.android.application.Collect;
 import org.odk.collect.android.logic.FormInfo;
 import org.odk.collect.android.provider.FormsProviderAPI.FormsColumns;
 import org.odk.collect.android.provider.InstanceProviderAPI.InstanceColumns;
 import org.odk.collect.android.storage.StoragePathProvider;
+import org.odk.collect.android.utilities.MediaUtils;
 
 public final class ContentResolverHelper {
```

Comment:
I think it'd be nice to change the name here to `...Utils` as that seems to have become idiomatic for static method buckets.

Reply:
You mean ContentResolverUtils? What about the package then? It's in helpers package along with other classes that use helper suffixes.

Reply:
Hmmm good point. I'd be up for moving those out into `utils` (or more appropriate packages) but now is maybe not the time.